### PR TITLE
Updates to migration topic (cmdline_migration.md)

### LINKED ISSUE #55: Hotspot / OpenJ9 commands comparison topic
**New topic**

For OpenJDK developers and users that are familiar with HotSpot options, we should carry a topic that helps people find the equivalent command in OpenJ9.

| HotSpot Option | OpenJ9 Option | Usage |
|:------------------|:-------------------|:-------|
| `-Xgc`                 | `-Xgcpolicy`        | Configuring your garbage collection policy |
| `-XX:ParallelGCThreads=<N>` |  `-Xgcthreads` | Configure number of GC threads |
| `-XX:+-UseNUMA` | `-Xnuma` | appear similar - investigation needed |
| `-XX:+DisableExplicitGC` | `-Xdisableexplicitgc` | Disables System.gc() calls |
| `-XX:-DisableExplicitGC` | `-Xenableexplicitgc` | Reenables System.gc() calls |
|*x* |*y*  | *For ..* |

Please add comments into this issue and I'll keep this top table up to date.

### PR DESCRIPTION
**Pull request by pmhayward**
Fixes #55

Signed-off-by: Peter Hayward <pmhayward@uk.ibm.com>

### CODE REVIEW

**Comment by pmhayward** [comment]
Jenkins doc stage

**Comment by pmhayward** [comment]
@DanHeidinga  @pshipton  Please review this migration topic (available on staging server at http://staging.eclipse.org/openj9/docs/cmdline_migration/) before we merge the PR. Note: this is work in progress - we know there is more to add! But want to make a start and check that what we have so far is correct. Thanks.

File: docs/cmdline_migration.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+* [1] https://www.gnu.org/software/classpath/license.html
+* [2] http://openjdk.java.net/legal/assembly-exception.html
+*
+* SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH
+* Classpath-exception-2.0 OR LicenseRef-GPL-2.0 WITH Assembly-exception
+-->
+
+# Migrating to OpenJ9
+
+If you are already familiar with HotSpot command-line options but want the advantages of OpenJ9, the following information will prove helpful.In all cases, check individual topics for minor discrepancies in the way these options might work.
```
Comment (pshipton) [inline docs/cmdline_migration.md]: Missing space after the period `helpful.In`

File: docs/cmdline_migration.md [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+| [`-Xbootclasspath`](xbootclasspath.md)                | Specifies the search path for bootstrap classes and resources.                             |
+| [`-Xcheck:jni`](xcheck.md)                            | Runs additional checks for JNI functions during VM startup.                                |
+| [`-Xfuture`](xfuture.md)                              | Turns on strict class-file format checks.                                                  |
+| [`-Xint`](xint.md)                                    | Runs an application in interpreted-only mode.                                              |
+| [`-Xmn`](xmn.md)                                      | Sets the initial and maximum size of the new area when using -Xgcpolicy:gencon.            |
+| [`-Xms`](xms.md)                                      | Sets the initial size of the heap.                                                         |
+| [`-Xmx`](xmn.md)                                      | Specifies the maximum size of the memory allocation pool. (Equivalent to -XX:MaxHeapSize.) |
```
Comment (pshipton) [inline docs/cmdline_migration.md]: The link is to the wrong topic.

This sets the maximum size of the object heap. "memory allocation pool" seems more generic. I guess "object memory allocation pool" would work.

Please clarify that OpenJ9 does not recognize -XX:ThreadStackSize

File: docs/cmdline_migration.md [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+| [`-XX:-DisableExplicitGC`](xxdisableexplicitgc)       | Reenables System.gc() calls                                                                |
+| [`-XX:MaxDirectMemorySize`](xxmaxdirectmemorysize.md) | Sets a limit on the amount of memory that can be reserved for all direct byte buffers.     |
+| [`-XX:-UseCompressedOops`](xxusecompressedoops.md)    | Disables compressed references in 64-bit JVMs. (See also -Xcompressedrefs)                 |
+
+- These Hotspot command-line options have equivalents in OpenJ9 that are not specified in the same way, but perform the same function:
+
+| HotSpot Option          | OpenJ9 Option                                  | Usage                                                      |
+|-------------------------|------------------------------------------------|------------------------------------------------------------|                                                                        
+| `-Xcomp`                | [`-Xjit:count=0`](xjit.md)**<sup>1</sup>**     | `-Xcomp` disables interpreted method invocations.          |
```
Comment (pshipton) [inline docs/cmdline_migration.md]: OpenJ9 understands -Xcomp and maps it to -Xjit:count=0. Did somebody request this option to be deprecated?

File: docs/cmdline_migration.md [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+| [`-Xint`](xint.md)                                    | Runs an application in interpreted-only mode.                                              |
+| [`-Xmn`](xmn.md)                                      | Sets the initial and maximum size of the new area when using -Xgcpolicy:gencon.            |
+| [`-Xms`](xms.md)                                      | Sets the initial size of the heap.                                                         |
+| [`-Xmx`](xmn.md)                                      | Specifies the maximum size of the memory allocation pool. (Equivalent to -XX:MaxHeapSize.) |
+| [`-Xnoclassgc`](xclassgc.md)                          | Disables class garbage collection (GC).                                                    |
+| [`-Xrs`](xrs.md)                                      | Prevents the OpenJ9 run time environment from handling signals.                            |
+| [`-Xss`](xss.md)                                      | Sets the thread stack size. (Equivalent to -XX:ThreadStackSize.)                           |
```
Comment (pshipton) [inline docs/cmdline_migration.md]: Please clarify that OpenJ9 does not recognize -XX:ThreadStackSize

**Comment by DanHeidinga** [comment]
@amicic Can you review the `-XX:+UseNUMA` related info?

File: docs/cmdline_migration.md (hunk 4 above)
Comment (pshipton) [reply docs/cmdline_migration.md]: Adding OpenJ9 support for -XX:ThreadStackSize is also covered by #2834

File: docs/cmdline_migration.md (hunk 2 above)
Comment (pmhayward) [reply docs/cmdline_migration.md]: Sorry... getting my -Xm's mixed up :)

**Pull request merged**